### DISCUSSION TITLE
Copy plugin changes back after stop

### DISCUSSION THREAD

**Discussion by JAPGamer** [itzg/docker-minecraft-server#2089] [Q&A]
Hey,

I want my server to copy back the plugins folder (with for example created config files) from the container /data/plugins back to my host ./plugins folder?

Does anyone know how can I achieve this?

Thanks.

Here is my docker-compose.yml
`version: "3"

services:
  mc:
    image: itzg/minecraft-server
    ports:
      - 25565:25565
    environment:
      EULA: "TRUE"
      VERSION: "1.19.2"
      TYPE: "PAPER"
    tty: true
    stdin_open: true
    restart: unless-stopped
    volumes:
      - "./plugins:/plugins"
`

**Comment by itzg** [comment]
It's not possible at that this point. Can you clarify why you would want to? Im also wondering why you're not mapping `/data` since that's where all the ongoing persistent data resides?
